static/js/topic_page/ranking_unit.tsx

```diff
@@ -29,18 +29,54 @@ const NUM_FRACTION_DIGITS = 2;
 export interface Point {
   placeDcid: string;
   placeName?: string;
-  stat: number;
+  value: number;
+  /**
+   * If not provided, the component will calculate the rank based on the order of the input points.
+   */
+  rank?: number;
 }
 
 interface RankingUnitPropType {
   title: string;
   points: Point[];
-  statVarName: string;
+  /**
+   * If true, rows are ordered from highest ranks to lowest, e.g., 1,2,3...
+   * Otherwise, from lowest to highest, e.g., n,n-1,n-2,...
+   */
+  isHighest: boolean;
+  statVarName?: string;
```

Comment:
Because we're making this a more generalized component, I think we should remove this optional `statVarName` prop and just have the `title` prop. We can do the `formatString()` in the RankingTile component and pass the fully formed title to this RankingUnit component

Reply:
Done